pkgs/servers/isso/default.nix

```diff
@@ -25,8 +37,16 @@ with python3Packages; buildPythonApplication rec {
 
   nativeBuildInputs = [
     cffi
+    nodejs
   ];
 
+  preBuild = ''
+    ln -s ${nodeDependencies}/lib/node_modules ./node_modules
+    export PATH="${nodeDependencies}/bin:$PATH"
+
+    make js
```

Comment:
Should we add -j$NIX_BUILD_CORES?

Reply:
I don't think it makes much sense here. The time spent in this build step is under 10s.

I tried it with adding `-j` which adds another 3s to the build time.